bin/git-overwritten

```diff
@@ -22,6 +22,9 @@ unset head
 unset base
 unset diff_type
 
+# Figure out what argument to use for date as GNU & BSD `date`s are different
+date --version 2>&1 >/dev/null && DATEARG='-d@' || DATEARG='-r'
```

Comment:
Works for me. The world is going to crumble anyway if BSD ever decides to implement GNU compatible flags and doesn't implement full feature parity for any given command, but there is no reason to contribute to that mess.